Dockerfile

```diff
@@ -0,0 +1,19 @@
+# syntax=docker/dockerfile:1.2
+
+# Copyright 2020-2021 Authors of Cilium
+# SPDX-License-Identifier: Apache-2.0
+
+ARG CILIUM_BUILDER_IMAGE=quay.io/cilium/cilium-builder:be4c0e477c4c8a61b3a41f6e8a0bab1d686ccd4e@sha256:0b69c4c6b680b6827c98167837a96647fa7d710912f2a001053b6c3bb0d4d9ae
+
+FROM ${CILIUM_BUILDER_IMAGE} as builder
+WORKDIR /go/src/github.com/cilium/cilium-cli
+COPY . .
+RUN make
+
+FROM bitnami/kubectl:1.16.15-debian-10-r220@sha256:2c35939baa0105d842f6ee144d81b083fba7b2d54f12d10025c243fa75a72a9b as kubectl
```

Comment:
regarding `kubectl`, we can download it directly from `https://dl.k8s.io/release/${k8s_version}/bin/linux/amd64/kubectl` where `k8s_version` is something that we can provide